apps/events.md

```diff
@@ -21,29 +21,42 @@ For more detailed information on a specific event, see the section [Examples](#e
 
 | Event | Description |
 | - |  - |
-| `pageview` | Is triggered when the customer opens or reloads a page. Informs about the path of the page. |
```

Comment:
Many changes in this section were taken over (and slightly adjusted to the Markdown format) from this PR: https://github.com/ePages-de/ng-api-docs/pull/277/files . But there are some differences as several new events were marked as BYD only, so I didn't include them in here. Please make sure that the changes actually also apply to the NOW docs and that I didn't miss changes :)

Reply:
All changes apply to both products. Just some events are only triggered by certain features that are not necessarily available in both products. The wishlist button is not available on byd so it only can trigger on Now/Unity. The checkout is only in beyond so it can not be triggered on Now/Unity.

Reply:
I see. Well if they can't be triggered for this product, I would not include them in the docs and keep it as it is.